ggshield/iac/iac_scan_models.py

```diff
@@ -37,34 +45,95 @@ class IaCDiffScanResult(Base):
     IaCDiffScanResult, BaseSchema
 )
 
+logger = logging.getLogger(__name__)
+
+
+class TmpGGClient(GGClient):
+    """Temporary class used until api_iac_diff_scan can be moved to Ggclient."""
 
-class MockClient(GGClient):
-    # TODO: move this into GGClient
-    def mock_api_iac_diff_scan(
+    def post(
```

Comment:
I think there is still a need to override those methods. The `post` method was override to fix the `data` send in `json`, I believe this is still not the case otherwise. The `request` method was override because of a typing error.

I change their name to avoid collision for now and I will inspect further to see if they can be removed.

Reply:
Py-gitguardian has been updated and those methods have been removed.